### PR TITLE
Updating to use new 'openshift-applier' repo

### PR DESCRIPTION
**Pull request by oybed**

### CODE REVIEW

**Comment by sherl0cks** [comment]
CI fails:

```
[labs-ci-cd] Running shell script
+ ansible-galaxy install -r requirements.yml --roles-path=roles
- extracting openshift-applier to /tmp/workspace/labs-ci-cd-ci-for-labs-ci-cd-pipeline/labs-ci-cd/roles/openshift-applier
- openshift-applier (v3.7.2) was installed successfully
[Pipeline] sh
[labs-ci-cd] Running shell script
+ ansible-playbook ci-playbook.yaml -i inventory/ -e 'project_name_postfix=-pr-152 scm_ref=pr-152'
ERROR! the role 'casl-ansible/roles/make-applier-projects-unique' was not found in /tmp/workspace/labs-ci-cd-ci-for-labs-ci-cd-pipeline/labs-ci-cd/roles:/home/jenkins/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/tmp/workspace/labs-ci-cd-ci-for-labs-ci-cd-pipeline/labs-ci-cd

The error appears to have been in '/tmp/workspace/labs-ci-cd-ci-for-labs-ci-cd-pipeline/labs-ci-cd/ci-playbook.yaml': line 16, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  roles:
    - casl-ansible/roles/make-applier-projects-unique
      ^ here
```

Looks like we need to update:
- https://github.com/rht-labs/labs-ci-cd/blob/master/local-role-test-playbook.yaml#L14
- https://github.com/rht-labs/labs-ci-cd/blob/master/ci-playbook.yaml#L15
- https://github.com/rht-labs/labs-ci-cd/blob/master/unique-projects-playbook.yaml#L9

**Comment by sherl0cks** [comment]
@oybed I just noticed we need to update https://github.com/rht-labs/labs-ci-cd/blob/master/run.sh#L37 if you want be consistent about `galaxy` dir

File: run.sh [hunk 1]
```diff
@@ -34,10 +34,10 @@ function printBanner(){
 ###########################
 printBanner
 
-ansible-galaxy install -r requirements.yml --roles-path=roles
+ansible-galaxy install -r requirements.yml --roles-path=galaxy
```
Comment (oybed) [inline run.sh]: @sherl0cks it's already here

**Comment by oybed** [comment]
@sherl0cks that's already part of this PR - see inline comment

File: run.sh (hunk 1 above)
Comment (sherl0cks) [reply run.sh]: oh dear me it is

**Pull request merged**